include/ddns.h

```diff
@@ -50,7 +50,7 @@
 #define DDNS_FORCED_UPDATE_PERIOD         (30 * 24 * 3600)        /* 30 days in sec */
 #define DDNS_DEFAULT_CMD_CHECK_PERIOD     1       /* sec */
 #define DDNS_DEFAULT_ITERATIONS           0       /* Forever */
-#define DDNS_HTTP_RESPONSE_BUFFER_SIZE    BUFSIZ  /* 8 kiB */
+#define DDNS_HTTP_RESPONSE_BUFFER_SIZE    2500    /* Bytes */
```

Comment:
This actually reduces the response buffer size from 8 kiB to 2.5kB on my Ubuntu 64-bit system.  Is it macOS that has BUFSIZ set to 1 kiB?

Reply:
> Is it macOS that has BUFSIZ set to 1 kiB?

That's where I saw it, yes. If this is Mac specific I'm happy to add an ifdef. Or how about something generic like this?

```
#define DDNS_HTTP_RESPONSE_BUFFER_SIZE	(BUFSIZ < 4096 ? 4096 : BUFSIZ)
```

Reply:
That works for me, although I guess I'd prefer having the same (8 kiB) on all major platforms.